models/seeds/02-books.js

```diff
@@ -13,11 +13,11 @@ exports.seed = function(knex, Promise) {
           synopsis:
             'Shoe Dog (2016) tells the story of the man behind the famous footwear company Nike. These blinks offer a peek into the mind of genius entrepreneur Phil Knight and detail the rollercoaster ride he went to through to build up his company.',
           cover_img:
-            'https://cdn1.imggmi.com/uploads/2019/8/19/31e08cd0fb2b8cef8a946c7ea4a28a0e-full.png',
+            'https://cdn1.imggmi.com/uploads/2019/8/22/a3cd260d27f5d97ff5b1380f283e6cc2-full.jpg',
```

Comment:
Does Messenger scale the image down to fit or does it just crop any extra off?

I just want to make sure we have a professional looking image to display to users (ie: no text cut off)

Reply:
It shows the whole book title with the swoosh